UN-OCHA/ocha_basic
# CD-24: frontend workflow to compliment OCHA styleguide

This PR supplies an almost identical workflow to the UN-OCHA/styleguide tools, allowing developers to easily switch between them and reuse commands, muscle memory, and workflows.

## Components

* `.nvmrc` to lock node.js to a specific version
* gulp.js workflow for Sass compiling
* GPL 2.0 license since it's a derivative of Drupal
* pre-commit hook to remove sourcemaps from compiled CSS

## Testing / Approval

1. Clone repository and check out this PR into an existing D7 installation
2. ensure you're using node v8.11.3
3. `npm install` to get new tools set up
4. Run `gulp dev` and see what happens
5. **Discuss:** is the included precommit git hook useful or getting in the way? checkout a new branch, make a small change and commit it to test the precommit hook.
